thepracticaldev/dev.to
# Fixed broken video capabilities of Connect

## What type of PR is this? (check all applicable)


## Description

I did some refactoring and lint fixes in here, but the main issue was that `fullscreen` was undefined. It's a prop that is passed in.

## Related Tickets & Documents

## QA Instructions, Screenshots, Recordings

Before

![image](https://user-images.githubusercontent.com/833231/85807737-c7f85a80-b720-11ea-8abf-3e0ab44e6c99.png)

After

![image](https://user-images.githubusercontent.com/833231/85808854-12c7a180-b724-11ea-908b-869330005d99.png)

To test:

* Go to any Connect channel and type in `/call` then press the ENTER key.
* A video session should begin.

## Added tests?


## Added to documentation?


## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![Snippet of a commercial for Kodak VHS and Beta tapes](https://media.giphy.com/media/Y3wfRUsgQ1TS9KJXFz/giphy.gif)
